src/use/use.py

```diff
@@ -928,7 +930,9 @@ def _use_str(
                 else:
                     try:
                         data = response.json()
-                        hit = find_latest_working_version(data["releases"])
+                        hit = find_latest_working_version(data["releases"], hash_algo=hash_algo)
```

Comment:
hash_algo is an enum member, it should be hash_algo.name to get the str
```suggestion
                        hit = find_latest_working_version(data["releases"], hash_algo=hash_algo.name)
```

Reply:
Addressed in commit #2